WordPress/WordPressTest/PostTests.swift

```diff
@@ -323,11 +323,11 @@ class PostTests: XCTestCase {
 
         revision.status = .pending
         let pendingStatusDisplay = "\(Post.title(for: .pending))"
-        XCTAssertEqual(revision.statusForDisplay(), String(format: NSLocalizedString("%@, %@", comment: ""), pendingStatusDisplay, local))
```

Comment:
This `"%@, %@"` format [is not itself translated in the `Post.swift` file and class](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Models/Post.swift#L284) that this tests (instead the 2 arguments are `NSLocalizedString`'d separately); so no reason to use `NSLocalizedString` for the format in the test itself.